server/middlewares/validations/UserVerification.js

```diff
@@ -0,0 +1,80 @@
+import jwt from 'jsonwebtoken';
+
+import db from '../../database/models';
+
+const { Article, Rating } = db;
+/**
+ *
+ */
+class UserVerification {
```

Comment:
Why this new file. Everything here has to do with rating an article why the name `UserVerification`

Reply:
I choose that name because I see the user as the key subject. There are two functions in the file/class: 
- verifyUserArticle: This one verifies that the Article belongs to the User.
- verifyUserRating: This one verifies that the specific Rating associated with the Article belongs to the User.

Also, I thought other functions that take the User as the key subject could be added to the file/class.